frontend/src/components/project_page/components/UserAvatar.tsx

```diff
@@ -7,9 +7,14 @@ export interface UserAvatarProps {
 
 const UserAvatar: FunctionComponent<UserAvatarProps> = ({ user }) => {
   return (
-    <div className="mr-1">
-      <img className="w-10" src={user.avatarUrl} alt={user.username} />
-    </div>
+    // <div className="mr-2 ml-2">
```

Comment:
If the image tag is the intended one to use, let's just remove the commented out div code.

Reply:
Removed commented out code